framework/db/ExpressionBuilder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace yii\db;
+
+/**
+ * Interface ExpressionBuilderInterface
+ *
+ * @author Dmitry Naumenko <d.naumenko.a@gmail.com>
+ * @since 2.0.14
+ */
+class ExpressionBuilder implements ExpressionBuilderInterface
+{
+    use ExpressionBuilderTrait;
+
+    /**
+     * {@inheritdoc}
+     * @param Expression|ExpressionInterface $expression the expression to be built
+     */
+    public function build(ExpressionInterface $expression, &$params = [])
+    {
+        $params = array_merge($params, $expression->params);
+        return $expression->__toString();
```

Comment:
I'd explicitly add a method to the interface instead of using magical `__toString`.

Reply:
This expression builder is specific for `yii\db\Expression`. Which kind of method would you add to the interface?

Reply:
Nevermind. It's fine.